source/secure_tunneling.c

```diff
@@ -1964,7 +1962,9 @@ int aws_secure_tunnel_service_operational_state(struct aws_secure_tunnel *secure
                 AWS_ZERO_STRUCT(frame_options);
                 frame_options.opcode = AWS_WEBSOCKET_OPCODE_PING;
                 frame_options.fin = true;
-                secure_tunnel->vtable->aws_websocket_send_frame_fn(secure_tunnel->websocket, &frame_options);
+                if (secure_tunnel->vtable->aws_websocket_send_frame_fn(secure_tunnel->websocket, &frame_options)) {
```

Comment:
error_code is being incorrectly initialized with AWS_OP_SUCCESS and not AWS_ERROR_SUCCESS